changelog.d/ast-autofix.fix

```diff
@@ -0,0 +1 @@
+Start to use the AST to render autofixes in some specific cases. As this is extended, autofix will become more correct and more powerful.
```

Comment:
I considered it, but in the end I don't feel like this fully addresses that issue. Sure, it fixes the one small test case given there, but if you tweak anything about it, it will no longer work. I'd prefer to have more cases filled in so that this is more general before closing that issue. I was really on the fence though, so if you prefer I can make this change.